src/main/java/net/openhft/chronicle/core/Jvm.java

```diff
@@ -175,7 +177,8 @@
         RESOURCE_TRACING = Jvm.getBoolean("jvm.resource.tracing");
 
         Logger logger = LoggerFactory.getLogger(Jvm.class);
-        logger.info("Chronicle core loaded from " + Jvm.class.getProtectionDomain().getCodeSource().getLocation());
+        if (logger.isInfoEnabled())
```

Comment:
do we really need to bother when this is only called once?

Reply:
No. This is a bit overdoing.